src/OmniSharp.Host/Startup.cs

```diff
@@ -36,18 +38,21 @@ public Startup(IOmniSharpEnvironment env)
 
             var configBuilder = new ConfigurationBuilder()
                 .SetBasePath(AppContext.BaseDirectory)
-                .AddJsonFile("config.json", optional: true)
+                .AddJsonFile(OmniSharpConstants.OmnisharpConfigFile, optional: true)
                 .AddEnvironmentVariables();
 
             if (env.OtherArgs?.Length > 0)
             {
                 configBuilder.AddCommandLine(env.OtherArgs);
             }
 
+            // Use the global omnisharp config if there's any in the shared path
+            configBuilder.CreateAndAddGlobalOptionsFile(env);
```

Comment:
the local file `{working_dir}/omnisharp.json` is not implicitly created, and users have to manually add it, so at least in that sense the behavior for the global one is consistent